src/Sidebar/Sidebar.tsx

```diff
@@ -56,40 +53,40 @@ const StyledLogo = styled.li`
   text-transform: uppercase;
   position: relative;
   top: 0;
-  margin-bottom: 15rem;
+  margin-bottom: 4rem;
   color: red;
 `;
 
 const StyledNavItem = styled.li`
```

Comment:
Changing this to a div should get rid of those dots.
```suggestion
const StyledNavItem = styled.div`
```